packages/angular_devkit/build_angular/src/webpack/plugins/karma/karma.ts

```diff
@@ -86,32 +86,11 @@ const init: any = (config: any, emitter: any) => {
   config.reporters.unshift('@angular-devkit/build-angular--event-reporter');
 
   // When using code-coverage, auto-add karma-coverage.
-  if (options.codeCoverage) {
-    config.plugins = config.plugins || [];
-    config.reporters = config.reporters || [];
-    const { plugins, reporters } = config;
-    const hasCoveragePlugin = plugins.some(isPlugin('karma-coverage', 'reporter:coverage'));
```

Comment:
I guess `isPlugin` function definition can be cleaned up as well?

Reply:
Now it's needed :)